CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## devel
 
+### Improvements
+#### Modules:
+  - zabbix_action - fixed error on changed API fields for Zabbix 5.0
```

Comment:
How about add at the end the related for PR or issue link?
e.g.

```
- zabbix_action - fixed error on changed API fields for Zabbix 5.0(https://github.com/ansible-collections/community.zabbix/pull/94)
```

What do you think @D3DeFi ?

Reply:
Oh I absolutely like that. Wasn't that even a standard/test/check in old repo?

Reply:
Thank you for an additional link of the issue.
When the Zabbix module was in Ansible upstream(2.9 and less), we wrote it in a fragmentation file.